.gitignore

```diff
@@ -39,4 +39,8 @@
 /yarn-error.log
 yarn-debug.log*
 .yarn-integrity
-/.idea/
\ No newline at end of file
+/.idea/
+/.history
+/test
+/Gemfile.lock

```

Comment:
shouldn't be in the .gitignore file: delete extra files in test and kepp Gemfile.lock same as github repo version

Reply:
ok